src/components/Message/MessageListItem.tsx

```diff
@@ -0,0 +1,29 @@
+interface MessageListItemProps {
+  user: User | null;
+}
+
+export default function MessageListItem({ user }: MessageListItemProps) {
+  return (
+    <>
+      <div className="p-3 hover:bg-secondary rounded-lg my-1 transition-colors group">
+        <div className="flex items-center justify-between ">
+          <div className="flex items-center justify-center">
+            <img
+              src={user?.image || "/logo.png"}
+              alt="프로필 이미지"
+              className="w-[30px] h-[30px] mr-3 mb-1 rounded-full"
```

Comment:
```suggestion
              className="w-[30px] h-[30px] mr-3 mb-1 rounded-full profile"
```

profile은 새로 머지된 PR에서 추가된 프로필 이미지에 공통으로 적용되는 스타일 입니다!